crates/ruff/src/rules/flake8_type_checking/snapshots/ruff__rules__flake8_type_checking__tests__runtime-import-in-type-checking-block_TCH004_8.py.snap

```diff
@@ -2,15 +2,5 @@
 source: crates/ruff/src/rules/flake8_type_checking/mod.rs
 expression: diagnostics
 ---
-- kind:
-    RuntimeImportInTypeCheckingBlock:
-      full_name: typing.Any
-  location:
-    row: 5
-    column: 23
-  end_location:
-    row: 5
-    column: 26
-  fix: ~
-  parent: ~
+[]
```

Comment:
This _shouldn't_ error because it's _not_ required at runtime, despite being on the RHS.